test/widgets/app_test.dart

```diff
@@ -288,4 +288,19 @@ void main() {
       check(findSnackBarByText('unrelated').evaluate()).single;
     });
   });
+    testWidgets('Home scrollable test, when contents do not fit to screen', (tester)async{
+      await tester.pumpWidget(MaterialApp(
+        home: Scaffold(body: Center(child: SingleChildScrollView(child: Column(
+        children: List.generate(15, (count)=>Text('Pick $count')),
+      ))))));
+      //make sure the added scrollable widget is functional
+      expect(find.byType(SingleChildScrollView), findsOneWidget);
+      expect(find.text('Pick 0'), findsOneWidget);
+
+      await tester.drag(find.byType(SingleChildScrollView), const Offset(0, -300));
+      //Test for items down the column
+      await tester.pump();
+      expect(find.text('Pick 14'), findsOneWidget);
+    });
```

Comment:
This is meant to test the code you added in `HomePage`, right? So it should exercise that code itself, instead of writing code for a new page that doesn't appear in the app. :)

To make it so the content runs longer than a screenful, try making the test simulate a shorter device height, and/or simulate an increased device text-size setting. We have examples of both in other widget tests.

(Similarly for `AddAccountPage`.)

Reply:
Thank you for pointing this out, I will make the changes and send a PR